importer/import_json.py

```diff
@@ -41,8 +41,17 @@
         "result",
         "oparl_id",
     ],
-    models.Consultation: ["meeting_id", "paper_id", "oparl_id"],
-    models.File: ["name", "oparl_download_url", "oparl_access_url", "oparl_id"],
+    models.Consultation: [
```

Comment:
meine-stadt-transparent uses [black](https://github.com/psf/black), I've added a ci job in 97fe124b8cfc3fd7a54ad0f46ca3cdf4b239b128 to lint for that